src/AuthorizationServiceInterface.php

```diff
@@ -25,12 +25,12 @@
      * This method is intended to allow your application to build
      * conditional logic around authorization checks.
      *
-     * @param \Authorization\IdentityInterface $user The user to check permissions for.
+     * @param \Authorization\IdentityInterface|null $user The user to check permissions for.
      * @param string $action The action/operation being performed.
      * @param mixed $resource The resource being operated on.
      * @return bool
      */
-    public function can(IdentityInterface $user, $action, $resource);
+    public function can($user, $action, $resource);
```

Comment:
Why removing? Can't you just ? nullable it?

Reply:
OK, true, wasn't aware of it, need to get used to the new php features for CakePHP :)

Reply:
Just bear in mind that this plugin then needs to add `>=7.1` in https://github.com/cakephp/authorization/blob/master/composer.json#L11

Reply:
@markstory your opinion on that?

Reply:
ok. for 4.0 then.